### PR TITLE
experimental

### PR DESCRIPTION
**Pull request by cyberliem**

### CODE REVIEW

File: core/types/transaction.go [hunk 1]
```diff
@@ -189,6 +203,10 @@ func isProtectedV(V *big.Int) bool {
 
 // EncodeRLP implements rlp.Encoder
 func (tx *Transaction) EncodeRLP(w io.Writer) error {
+	if (tx.data.Provider == nil) && (tx.data.Owner == nil) {
```
Comment (ducquangkstn) [inline core/types/transaction.go:206]: if PR, PS, PV is not null then this is also not normal transaction

File: core/types/transaction_test.go [hunk 2]
```diff
@@ -51,6 +53,14 @@ var (
 	)
 )
 
+func TestTransactionCompatiblity(t *testing.T) {
+	etherHashString := "0x32d0ec31372e18e22e4af25a00926e344c16473978dce150112a89d0bdf5795a"
+	evrTx := NewTransaction(0, common.HexToAddress("095e7baea6a6c7c4c2dfeb977efac326af552d87"), big.NewInt(1000), 0, big.NewInt(100), common.FromHex("123"))
```
Comment (ducquangkstn) [inline core/types/transaction_test.go:58]: It will be better if this is a signed transaction

File: core/types/transaction.go (hunk 1 above)
Comment (ducquangkstn) [reply core/types/transaction.go:206]: Also pls add comment to explain why we need to do this

**Comment by ducquangkstn** [comment]
Because the evrynet 's normal transaction encoding is the same as ethereum, we need to roll back to value at [here](https://github.com/Evrynetlabs/evrynet-node/blob/60d51554a3139828b642bda5a889288752f1a470/core/types/transaction_test.go#L79) and [here](https://github.com/Evrynetlabs/evrynet-node/blob/41891baf0e11977ae457969b2829b0c2aa2dd1fc/core/types/block_test.go#L31)

File: core/types/transaction_test.go (hunk 2 above)
Comment (devnmhoang) [reply core/types/transaction_test.go:60]: He signed this Tx on line 65.

**Pull request merged**